python/cudf/cudf/core/series.py

```diff
@@ -1598,6 +1598,15 @@ def dtype(self):
         """The dtype of the Series."""
         return self._column.dtype
 
+    @property  # type: ignore
+    @_performance_tracking
+    def dtypes(self):
+        """The dtype of the Series.
+
+        This is an alias for `Series.dtype`.
+        """
+        return self.dtype
```

Comment:
Turns out that Pandas uses `Series.dtypes` as an alias for `Series.dtype` (and some recent dask-expr code takes advantage of the Sereies<->DataFrame portability of the `dtypes` attribute).